package.json

```diff
@@ -11,8 +11,10 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "git": "^0.1.5",
```

Comment:
I dont think we need this dependency, please uninstall it:
`npm uninstall git`
and test the application